directory/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - directory
 
+## 1.14.1-beta.1 / 2022-06-13
```

Comment:
When merging this into `master`, make sure to remove any changes from the beta releases.